.travis.yml

```diff
@@ -161,10 +161,6 @@ script:
 #Build weston and weston plugin
 - git clone -b iahwc-plugin https://github.com/intel/external-weston.git ../weston
 - cd ../weston
-#HACK: Weston's paths are hardcoded to ../iahwc/..
-- cp -r ../IA-Hardware-Composer ../iahwc
-- cd -
-- os/linux/weston/build_script.sh --weston-dir=../weston --iahwc-dir=$(pwd) --build
```

Comment:
Doing this will disable the testing of weston and iahwc backend for weston.
The HACK line was written just as a hint that the weston's build system expects that iahwc is placed at ../iahwc relative to the build directory.
IIRC the build failure was due to compilation error in libva, which occurs much before this.